File: AddTwentyArray.java

import java.util.*;
class AddTwentyArray {
    
    public static void main(String[] args) {
    Scanner sc = new Scanner(System.in) ;
    int result = 0 ;
       int[] array = new int[20]; 
       
       System.out.println("enter 20 values. :");
       
       for(int i =0;i<20;i++){
           array[i] = sc.nextInt();
       }
       
      for(int i =0;i<20;i++){
           
           result += array[i];
       }
       
       System.out.println("The sum of array values. : "+result);
       
       
    }
}

File: AddTwentyArray.cpp
#include <iostream>
using namespace std;

int main() {
    int result = 0;
    int array[20];

    cout << "Enter 20 values: " << endl;

    for (int i = 0; i < 20; i++) {
        cin >> array[i];
    }

    for (int i = 0; i < 20; i++) {
        result += array[i];
    }

    cout << "The sum of array values: " << result << endl;

    return 0;
}
